File: AudienceAI - Android/app/src/main/java/com/exuberant/audienceai/Status.kt
package com.exuberant.audienceai


import com.google.gson.annotations.SerializedName

data class Status(
    @SerializedName("emotion_scores")
    val emotionScores: Any,
    @SerializedName("emotion_scores_json")
    val emotionScoresJson: String,
    @SerializedName("file_name")
    val fileName: String,
    @SerializedName("file_path")
    val filePath: String,
    @SerializedName("num_frames")
    val numFrames: Int,
    @SerializedName("status")
    val status: String
)

File: AudienceAI - Backend/audienceai-backend-master/pkg/entities/status.go
package entities

type Status struct {
	FilePath          string      `json:"file_path"`
	FileName          string      `json:"file_name" gorm:"primary_key"`
	Status            string      `json:"status"`
	EmotionScoresJson string      `json:"emotion_scores_json"`
	NumFrames         int         `json:"num_frames"`
	EmotionScores     [][]float64 `json:"emotion_scores" gorm:"-"`
}

type EmotionData struct {
	Data [][]float64 `json:"data"`
}
